operator/konflux.bundle.Dockerfile

```diff
@@ -14,6 +14,21 @@ RUN make bundle-post-process
 
 FROM scratch
 
+# Enterprise Contract labels.
+LABEL com.redhat.component="rhacs-operator-bundle-container"
+LABEL description="Operator Bundle Image for Red Hat Advanced Cluster Security for Kubernetes"
+LABEL distribution-scope="public"
+LABEL io.k8s.description="Operator Bundle Image for Red Hat Advanced Cluster Security for Kubernetes"
+LABEL name="rhacs-operator-bundle"
+# Release label is required by EC although has no practical semantics.
+# We also set it to not inherit one from a base stage in case it's RHEL or UBI.
+LABEL release="1"
+LABEL url="https://catalog.redhat.com/software/container-stacks/detail/60eefc88ee05ae7c5b8f041c"
+LABEL vendor="Red Hat, Inc."
+LABEL version="${MAIN_IMAGE_TAG}"
```

Comment:
Does `LABEL version="${MAIN_IMAGE_TAG}"` without declaring `ARG MAIN_IMAGE_TAG` in this stage? I believe it doesn't after inspecting labels of `quay.io/rhacs-eng/stackrox-operator-bundle:v4.6.0-167-g2e748850de-fast`.